tienvx/mbt-bundle/1.0/config/packages/tienvx_mbt.yaml

```diff
@@ -0,0 +1,5 @@
+tienvx_mbt:
+    max_path_length: 300
```

Comment:
No need to add these default values.

Reply:
Thanks for your reviewing. I removed tienvx/mbt-bundle. Please take a look again.